### PR TITLE
Fluently integrate search-combobox component as addon for input-field component 

### LINKED ISSUE #47: input-field looks bad if we use a combobox in unit field
image

### PR DESCRIPTION
**Pull request by ozanmuyes**
Fixes #47

### CODE REVIEW

**Comment by ceremcem** [comment]
@ozanmuyes  Any screenshots?

**Comment by ozanmuyes** [comment]
screenshot - 02082017 - 07 21 53 pm

File: src/client/components/input-field/index.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+.input-group-addon2:last-child {
+    /*.input-group-addon:last-child {*/
+        border-left: 0;
+    /*}*/
+}
+
+
+/* CEVEAT1: Children under `input-group-addon2` classed tag, won't render like as if they children of `input-group-addon` */
+.input-group-addon2>*:not(.input-group-with-selectize) {
```
Comment (ceremcem) [inline src/client/components/input-field/index.css]: Are we supposed to edit this component's css whenever we introduce a new component that might be used in `unit` field?

**Comment by ceremcem** [comment]
Closing as #47 is closed.

**Pull request closed without merging**